methods.go

```diff
@@ -83,3 +70,34 @@ func EnableColor(flag bool) {
 func ListLoggersNames() []string {
 	return []string{prettylogger.LoggerName, zaplogger.LoggerName, nonelogger.LoggerName}
 }
+
+// InitOtel configures OpenTelemetry to export data to OTEL_COLLECTOR_SVC using uptrace collector.
+// You have to set the env variable OTEL_COLLECTOR_SVC to enable otel.
+// It is required to call ShutdownOtel on the context at the end of the main.
+func InitOtel(log helpers.ILogger, service, version string) context.Context {
+	ctx := context.Background()
+	if svc, ok := os.LookupEnv("OTEL_COLLECTOR_SVC"); ok {
+		var attrs []attribute.KeyValue
+		if id, ok := os.LookupEnv("ACCOUNT_ID"); ok {
+			attrs = append(attrs, attribute.String("account.id", id))
+		}
+		dsn := url.URL{
+			Scheme: "http",
+			User:   url.User("t"),
+			Host:   svc,
+			Path:   "1",
+		}
+		uptrace.ConfigureOpentelemetry(
+			uptrace.WithDSN(dsn.String()),
+			uptrace.WithServiceName(service),
+			uptrace.WithServiceVersion(version),
+			uptrace.WithResourceAttributes(attrs...),
```

Comment:
question: I notice we use the Uptrace SDK.

Would it be possible to swap out the telemetry platform without swapping out the SDK? If not, why are we using the Uptrace SDK, and not the official OTel one?

Reply:
the uptrace client is just a wrapper on otel, but I wanted to use it because it's much simpler to configure
it is compatible with regular otel, as a proof we send data to an otel collector
then the collector can send telemetry data to several different backends

Reply:
Makes sense, thank you for explaining.